libs/RFIDKit/include/MagicCard.h

```diff
@@ -30,6 +30,8 @@ struct MagicCard {
 		return both;
 	}
 
+	[[nodiscard]] constexpr auto getLanguage() const -> uint8_t { return _tag.data.at(language_byte_index); }
```

Comment:
not asking for a change, but after re-reading the whole code, having at `getXXX` feels a bit verbose. It's just a simple struct, `card.id()` or `card.language()` would be good enough.

to keep in mind for a future refactor ;)

Reply:
To make it more readable and using Language inside MagicCard's architecture, use `uint16_t _id` & `Language _language` as variables instead of `rfid::Tag _tag`.